tests/containers/volumes.pm

```diff
@@ -40,7 +40,7 @@ sub run {
     my $test_container = "test_container";
     my $test_volume = "test_volume";
 
-    assert_script_run("mkdir $test_dir");
+    assert_script_run("rm -rf $test_dir ; mkdir $test_dir");
```

Comment:
```suggestion
    assert_script_run("mkdir -p $test_dir");
```
`-p` doesn't fail if the directory already exists

Reply:
Ignore this comment, better to nuke the directory and the contents therein as well.